source-control/readme.md

```diff
@@ -30,6 +30,11 @@ The team should at least be doing the following:
   * [README.md](../resources/templates/README.md)
   * [CONTRIBUTING.md](../resources/templates/CONTRIBUTING.md)
 
+For most engagements having a single hosted DevOps environment (i.e AzureDevOps) is the preferred path but there are times when a mixed DevOps environment (i.e. AzureDevOps for Agile/Work item tracking & GitHub for Source Control) is needed due to customer requirements. When working in a mixed environment:
```

Comment:
Should this have a title?
Also -- please coordinate with @rozele as he is making major changes to this doc - so that the changes don't get lost